### ISSUE TITLE
Cookie expiration

### ISSUE THREAD

**Issue by hector6872** [Schlomoh/cookieConsent#8]
I don't know if it's on purpose but the cookie expiration is set as a _session cookie_. Browsers define when the "current session" ends and at least on Android it happens constantly.

**Comment by Schlomoh** [comment]
@hrules6872 good catch! I'm currently not able to actively maintain this repo, but am grateful for every contribution :) Feel free to modify things and create a PR
